src/theme/AddToMetaMaskButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import type { MetaMaskInpageProvider } from "@metamask/providers";
+
+declare global {
+  interface Window {
+    ethereum?: MetaMaskInpageProvider;
+  }
+}
+
+export default function AddToMetaMaskButton() {
```

Comment:
Can you add the corresponding module declaration with the `@theme` prefix to the `types.d.ts` file?

Reply:
I fixed the type